wiki/Beatmap_ranking_procedure/en.md

```diff
@@ -38,7 +38,9 @@ It is recommended to receive mods before requesting nominations from BNs, howeve
 
 ## Qualification
 
-Beatmaps that receive two nominations per each available game mode are moved to the [Qualified](/wiki/Beatmap/Category#qualified) beatmap category. Qualified beatmaps have score leaderboards, but do not reward players with [performance points](/wiki/Performance_points). When a beatmap stays in Qualified for at least 7 days, the [ranking queue](Ranking_queue) is able to move it to [Ranked](#ranked).
+Beatmaps that receive two nominations for the primary game mode are moved to the [Qualified](/wiki/Beatmap/Category#qualified) beatmap category[^hybrid-sets]. Qualified beatmaps have score leaderboards, but do not reward players with [performance points](/wiki/Performance_points). When a beatmap stays in Qualified for at least 7 days without any open [Problems or Suggestions](/wiki/Modding#types-of-mod-posts), the [ranking queue](Ranking_queue) is able to move it to [Ranked](#ranked).
+
+If a beatmap is disqualified, its ranked timer will be reset to 7 days if either new difficulties are added or if the beatmap is qualified with different Beatmap Nominators. Additionally, for every week a beatmap stays disqualified, the timer will increase by 1 day, up to a maximum of 14 days.
```

Comment:
`different Beatmap Nominators` implies that it resets when all nominators are different, i'd look into clarifying that any bn change causes the reset